Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/PersonDetailsFragmentTest.kt

```diff
@@ -172,12 +185,32 @@ class PersonDetailsFragmentTest : BaseUITest() {
     private fun boosterCertificateData(isCwa: Boolean = false): LiveData<List<CertificateItem>> = MutableLiveData(
         mutableListOf<CertificateItem>().apply {
             val vaccinationCertificate1 = mockVaccinationCertificate(number = 3, final = false, booster = true)
+
             val personCertificates = PersonCertificates(
                 listOf(vaccinationCertificate1),
                 isCwaUser = isCwa
             )
 
             add(PersonDetailsQrCard.Item(vaccinationCertificate1, false) {})
+
+            add(
+                VaccinationInfoCard.Item(
+                    vaccinationStatus = VaccinatedPerson.Status.BOOSTER_ELIGIBLE,
+                    daysUntilImmunity = null,
+                    boosterRule = mockk<DccValidationRule>().apply {
+                        every { identifier } returns "AH-1994"
+                        every { description } returns listOf(
+                            mockk {
+                                every { description } returns "Aufgrund Ihres vorhanden Impfzertifikats könnte für Sie eine Auffrischungsimpfung erforderlich sein, da Ihr letztes Impfzertifikat älter als 6 Monate ist"
+                                every { languageCode } returns "DE"
```

Comment:
what about `EN` ? screenshots are in `DE` , `EN`

Reply:
adjusted